api/app/controllers/spree/api/v2/base_controller.rb

```diff
@@ -43,7 +43,7 @@ def find_spree_current_order
             store:    spree_current_store,
             user:     spree_current_user,
             token:    order_token,
-            currency: params[:currency] || current_currency
+            currency: current_currency
```

Comment:
dont we lose flexibility, when removing custom currency passed to params?

Maybe in such case it makes sense to move `params[:currency]` to `BaseController#current_currency`?

Reply:
@VygovskySergey we need to validate if the given Store supports the desired currency, it should be expanded later, but giving an option for any currency would lead to really strange scenarios ;)